main/Helper.cpp

```diff
@@ -478,7 +478,7 @@ double ConvertTemperature(const double tValue, const unsigned char tSign)
 	return RoundDouble(ConvertToFahrenheit(tValue),1);
 }
 
-std::vector<std::string> ExecuteCommandAndReturn(const std::string &szCommand)
+std::vector<std::string> ExecuteCommandAndReturn(const std::string &szCommand, int *returncode)
```

Comment:
Ehm, that will not work? Then it would be better to do it this way:
int ExecuteCommandAndReturn(const std::string &szCommand, std::vector<std::string> szResult)
return returncode

??